freqtrade/exchange/exchange.py

```diff
@@ -750,7 +750,7 @@ def _get_stake_amount_limit(
         return self._get_stake_amount_considering_leverage(
             max(stake_limits) * amount_reserve_percent,
             leverage or 1.0
-        ) if isMin else min(stake_limits)  # TODO-lev: Account 4 max_reserve_percent in max limits?
```

Comment:
@xmatthias You said this wasn't needed right?

Reply:
no, max_reserve_percent for sure not.